dev/tasks/macros.jinja

```diff
@@ -221,3 +222,79 @@ on:
         cp ${formula} $(brew --repository homebrew/core)/Formula/
       done
 {% endmacro %}
+
+{%- macro change_r_pkg_version(is_fork, version = '\\2.\'\"$(date +%Y%m%d)\"\'' ) -%}
```

Comment:
Could you prefix the macros with `github_` so it's clear that they only work on GHA.